static/bidder-params/adview.json

```diff
@@ -0,0 +1,18 @@
+{
+  "$schema": "http://json-schema.org/draft-04/schema#",
+  "title": "Adview Adapter Params",
+  "description": "A schema which validates params accepted by the adview adapter",
+  "type": "object",
+  "properties": {
+    "placementId": {
+      "type": "string",
+      "description": "An ID which identifies the placement selling the impression"
```

Comment:
Does it make sense to also include a minLength of 1 for the placementId?

Reply:
yes, added.